locales/en-US/webextension.properties

```diff
@@ -0,0 +1,27 @@
+addonDescription = Page Shot takes clips and screenshots from pages, and can save a permanent copy of a page.
+addonAuthorsList = Ian Bicking, Donovan Preston, and Bram Pitoyo
+toolbarButtonLabel = Take a shot
+contextMenuLabel = Create Page Shot
+myShotsLink = My Shots
+screenshotInstructions = Drag or click on the page to select a region. Press ESC to cancel.
+saveScreenshotSelectedArea = Save
+saveScreenshotVisibleArea = Save visible
+saveScreenshotFullPage = Save full page
+cancelScreenshot = Cancel
+downloadScreenshot = Download
+notificationLinkCopiedTitle = Link Copied
+# LOCALIZATION NOTE(notificationLinkCopiedDetails): The string "{meta_key}-V" should be translated
```

Comment:
You can avoid the LOCALIZATION NOTE part, since tools will associate and display the comment to the following string.

Possible comment:
```
# The string "{meta_key}-V" should be translated to the region-specific
# shorthand for the Paste keyboard shortcut. {meta_key} is a placeholder for the
# modifier key used in the shortcut (do not translate it): for example, Ctrl-V
# on Windows systems.
```